submissions/m-ruslan/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="app.css" />
+    <link rel="icon" type="image/png" href="./img/cinema.png" />
+    <title>Document</title>
+</head>
+
+<body>
+
+    <header>
+        <img class="header__logoImg" src="./img/cinema.png" alt="Best Cinema Logo">
```

Comment:
The logo must be a link to the home page.

Reply:
Thank you. Due to we have only one page I've added the blank link.